### PR TITLE
added button to create desktop entry

### PR DESCRIPTION
**Pull request by akashdakoor**
Creates YouTube.desktop file in "~/.local/share/applications"

### CODE REVIEW

**Comment by shreyasbapat** [comment]
See the folder dist! There is an executable. We just have to create a desktop entry for that. Because for a python file to run, we would need to ask user to Install requirements

**Comment by shreyasbapat** [comment]
@akashdakoor Any updates?

**Comment by akashdakoor** [comment]
Edited the file!

File: YouTube.py [hunk 1]
```diff
@@ -26,7 +28,20 @@ def ondown():
     stream = video.streams.first()
     print(os.getcwd())
     stream.download()
-    
+
+def desk():
+    print(file_path)
+    entry = "[Desktop Entry] \nName=YouTube Downloader \nVersion=1.0 \n" \
+            "Exec=file_path + "/YouTube \n" \
+            "Icon=/usr/share/pixmaps/python.xpm \nComment=Download YT videos \n" \
```
Comment (shreyasbapat) [inline YouTube.py]: We would need to design a icon for the app!

File: YouTube.py [hunk 2]
```diff
@@ -40,5 +55,6 @@ def func(event):
 Button(root, text='View', command=onok).pack(side=TOP)
 Button(root, text='Download', command=ondown).pack(side=LEFT)
 Button(root, text='CLOSE',fg="red",command=root.destroy).pack(side= RIGHT)
+Button(root, text='DesktopEntry', command=desk).pack(side=BOTTOM)
```
Comment (shreyasbapat) [inline YouTube.py]: Amazing!

**Comment by shreyasbapat** [comment]
Well, it breaks the app. Did you try to run it?

**Comment by shreyasbapat** [comment]
I am reverting this commit :(

**Pull request merged**